README.md

```diff
@@ -140,7 +140,10 @@ Who Uses Nomad
 Contributing to Nomad
 --------------------
 
-If you wish to contribute to Nomad, you will  need [Go](https://www.golang.org) installed on your machine (version 1.10.2+ is *required*).
+If you wish to contribute to Nomad, you will need [Go](https://www.golang.org)
+installed on your machine (version 1.10.2+ is *required*).
+
+See the [`docs`](docs/) directory for more developer documentation.
```

Comment:
I would consider adding having them in `/contributors` or `/engineering` subtree to avoid confusing them with user docs.